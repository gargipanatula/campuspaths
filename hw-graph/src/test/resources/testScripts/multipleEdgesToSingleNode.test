# Create a graph
CreateGraph graph1

# Add nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4

# Add an edge
AddEdge graph1 n1 n4 e1
AddEdge graph1 n2 n4 e2
AddEdge graph1 n3 n4 e3
AddEdge graph1 n4 n4 e4

# Print the children of n4 in graph1
ListChildren graph1 n4