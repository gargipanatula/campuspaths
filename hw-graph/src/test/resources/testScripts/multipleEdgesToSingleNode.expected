# Create a graph
created graph graph1

# Add nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1

# Add an edge
added edge e1 from n1 to n4 in graph1
added edge e2 from n2 to n4 in graph1
added edge e3 from n3 to n4 in graph1
added edge e4 from n4 to n4 in graph1

# Print the children of n4 in graph1
the children of n4 in graph1 are: n4(e4)